docs/hub/main.md

```diff
@@ -11,6 +11,7 @@ We are helping the community work together towards the goal of advancing Artific
 
 Not one company, even the Tech Titans, will be able to “solve AI” by themselves – the only way we'll achieve this is by sharing knowledge and resources. On the Hugging Face Hub we are building the largest collection of models, datasets and metrics in order to democratize and advance AI for everyone 🚀. The Hugging Face Hub works as a central place where anyone can share and explore models and datasets.
 
+> For more detailed API documentation, you can find it [here](https://github.com/huggingface/huggingface_hub/tree/main/src/huggingface_hub) on the GitHub
```

Comment:
I'm not sure we want to add this. This site works as general Hub documentation. Most people are not looking for our `huggingface_hub` client library in here.

Reply:
I'd rather add it to documentation pages that leverage `hfh`, like https://github.com/huggingface/huggingface_hub/blob/main/docs/hub/how-to-downstream.md and https://github.com/huggingface/huggingface_hub/blob/main/docs/hub/how-to-upstream.md

Even https://github.com/huggingface/huggingface_hub/blob/main/docs/hub/adding-a-model.md, as it should be updated to include `huggingface_hub` instructions as well as raw git operations.

Reply:
> be updated to include huggingface_hub instructions

 just mention that one can use `huggingface_hub` to programmatically push a repo and link to a relevant page?

(more users just use git than huggingface_hub so I'd rather keep that as the prominent way to add a model)

Reply:
Will add updating those three with a snippet instead onto the TODO, and remove the direct link